cosmos/airflow/graph.py

```diff
@@ -142,7 +142,7 @@ def build_airflow_graph(
     a single test task will be added after the Cosmos leave tasks, and it is named using `dbt_project_name`.
     Finally, if the `test_behaviour` is `after_each`, a test will be added after each model.
 
-    If `emit_datasets` is True, tasks will create outlets using:
+    If `emit_datasets_from` is defined, tasks will create outlets using:
```

Comment:
This seems like a breaking change, we should keep the old parameter and add a deprecation warning since we aim to release this as part of 1.1 (minor release).